rpc/core/txs.go

```diff
@@ -91,27 +91,27 @@ func SignTx(tx types.Tx, privAccounts []*account.PrivAccount) (*ctypes.ResponseS
 		sendTx := tx.(*types.SendTx)
 		for i, input := range sendTx.Inputs {
 			input.PubKey = privAccounts[i].PubKey
-			input.Signature = privAccounts[i].Sign(sendTx)
+			input.Signature = privAccounts[i].Sign(config.GetString("chain_id"), sendTx)
```

Comment:
maybe this should be passed into the rpc call, and defaults to the config if empty.